### ISSUE TITLE
Feature request: Mode Place only where you look

### ISSUE THREAD

**Issue by FurnyGo** [aleksilassila/litematica-printer#71]
![image](https://user-images.githubusercontent.com/68079109/157424622-adcfb21e-b8cb-4ce8-8450-b2a0e7cda607.png)
Mode for servers with good anticheat, which won't let you place block where you are not looking

**Comment by white-devil42** [comment]
thats not why anticheats kick you,
its probably because of "air place",
he said he'll try to fix it

**Comment by FurnyGo** [comment]
> thats not why anticheats kick you, its probably because of "air place", he said he'll try to fix it

No, mod can place blocks on ait when i look on him on air, but sometimes it tries place behind me, and this dont work.

**Comment by white-devil42** [comment]
can u give server ip?

**Comment by FurnyGo** [comment]
> can u give server ip?

Why are you asking? It is pay-to-win server, more pay - less anticheat detection

**Comment by FurnyGo** [comment]
Ok I find easyPlaceMode feature in original litematic. Now I waiting for the update, when printer will rotate head to block that is being placed (something similar to SalHack 1.12.2 Printer, Inertia Notebot/Nuker)
